### DISCUSSION TITLE
<title> About Upgrading To Github Pro

### DISCUSSION THREAD

**Discussion by jackutea** [community/community#63156] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Github Pro Pricing does not list follow informations:
1. Limit of Repository storage size
2. Limit of LFS storage size

**Comment by vincentcui-ui** [comment, marked as answer]
Hey @jackutea,

Just chiming in on your question about GitHub Pro storage limits! 🚀

Repository storage size: There's a soft cap of 100 GB for each repository. If you exceed that, GitHub will send a friendly reminder to the repository owner. While pushing to oversized repositories is technically possible, it's not recommended since it might hamper performance.
LFS (Large File Storage) details: Individual LFS objects are capped at 2 GB. For the collective LFS storage, free accounts get about 1 GB of Git LFS storage. For GitHub Pro and above (like Team and Enterprise tiers), once you max out your included storage, you'd typically need to grab some additional data packs.
